src/v1/QueryManager.sol

```diff
@@ -19,16 +19,19 @@ abstract contract QueryManager is IQueryManager, Groth16VerifierExtension {
     uint256 public constant MAX_QUERY_RANGE = 50_000;
 
     /// @notice A constant gas fee paid for each request to reimburse the relayer when it delivers the response
-    uint256 public constant ETH_GAS_FEE = 0.01 ether;
-    uint256 public constant OP_GAS_FEE = 0.001 ether;
-    uint256 public constant CDK_GAS_FEE = 0.001 ether;
+    uint256 public immutable GAS_FEE;
+    uint256 private constant ETH_GAS_FEE = 0.01 ether;
+    uint256 private constant L2_GAS_FEE = 0.001 ether;
```

Comment:
I consolidated the chain fees into L2 vs "not L2", since they were all the same fee anyway.